package.json

```
@@ -43,37 +43,43 @@
                     "type": "string",
                     "default": "nvim",
                     "title": "Neovim executable path on Windows",
-                    "markdownDescription": "Path to Nvim executable if running VS Code on Windows. \n- Example: `C:\\tools\\neovim\\Neovim\\bin\\nvim.exe` (if you installed using Chocolatey)\n- If you want to use Windows Subsystem for Linux, enable the `useWSL` setting and set `neovimExecutablePaths.linux` instead"
+                    "markdownDescription": "Path to Nvim executable if running VS Code on Windows. \n- Example: `C:\\tools\\neovim\\Neovim\\bin\\nvim.exe` (if you installed using Chocolatey)\n- If you want to use Windows Subsystem for Linux, enable the `useWSL` setting and set `neovimExecutablePaths.linux` instead",
+                    "ignoreSync": true
```

for each case, would you mind moving the added line up so that the existing lines aren't changed. that makes the diff less noisy.
```suggestion
                    "ignoreSync": true,
                    "markdownDescription": "Path to Nvim executable if running VS Code on Windows. \n- Example: `C:\\tools\\neovim\\Neovim\\bin\\nvim.exe` (if you installed using Chocolatey)\n- If you want to use Windows Subsystem for Linux, enable the `useWSL` setting and set `neovimExecutablePaths.linux` instead"
```

---

Good idea !

Will do it if `ignoreSync` is preferable to the machine scope